packages/web3-utils/CHANGELOG.md

```diff
@@ -173,4 +173,9 @@ Documentation:
 
 ### Added
 
--   Add `isContractInitOptions` method (#6455)
\ No newline at end of file
+-   `SocketProvider` now contains public function `getPendingRequestQueueSize`, `getSentRequestsQueueSize` and `clearQueues` (#6451)
+-   Added `safeDisconnect` as a `SocketProvider` method to disconnect only when request queue size and send request queue size is 0 (#6451) 
```

Comment:
We need the PR number rather than the issue number.

```suggestion
-   `SocketProvider` now contains public function `getPendingRequestQueueSize`, `getSentRequestsQueueSize` and `clearQueues` (#6479)
-   Added `safeDisconnect` as a `SocketProvider` method to disconnect only when request queue size and send request queue size is 0 (#6479) 
```